.python-version

```diff
@@ -0,0 +1 @@
+pydicom
```

Comment:
Did you mean to add this? As far as I understand, this is the name of your virtual environment?

Reply:
Whoops, nope. Removed in [c37939e](https://github.com/pydicom/pydicom/pull/2148/commits/c37939eae18b7872a1d0c3ba5236ef17a50bfca1)